ext/opcache/Optimizer/block_pass.c

```diff
@@ -115,6 +115,11 @@ static int get_const_switch_target(zend_cfg *cfg, zend_op_array *op_array, zend_
 		/* fallback to next block */
 		return block->successors[block->successors_count - 1];
 	}
+	if ((opline->opcode == ZEND_MATCH_LONG && Z_TYPE_P(val) != IS_LONG)
+			|| (opline->opcode == ZEND_MATCH_STRING && Z_TYPE_P(val) != IS_STRING)) {
+		/* fallback to next block */
```

Comment:
The copied comment should be changed, the other referred to a less efficient **fallback** to the next block, and the next block was the start of the less efficient IS_EQUAL/JMPNZ chain

```suggestion
		/* This will always jump to the default arm */
```